src/core/engines/Cesium/Feature/Model/index.tsx

```diff
@@ -78,7 +79,7 @@ export default function Model({ id, isVisible, property, geometry, layer, featur
       featureId={feature?.id}
       draggable>
       <ModelGraphics
-        uri={model}
+        uri={model || url}
```

Comment:
The below is just something to keep in mind as you develop. I am assuming the user will be able to choose a TYPE (model or URL) from the right panel. If that isn't the case, ignore the rest of this message.

I've seen issues before while using Re:Earth that are caused by code like this. When a user can choose a type (ie. marker can be icon or point, this Model can be model or url, etc), if the user starts with model, and then switches to URL, or vice versa, both fields will have a value in the DB. Which means, even if the user selects a url type, if there is a model it'll show that.

Because of this, keep in mind that when we query for the model, we will need to either:

- Assign null/undefined after querying based on selected type(aka. If the type is url, make model null/undefined, if the type is model, make the url null/undefined.
OR
- Make sure the query conditionally fetches the values (see [directives](https://www.apollographql.com/docs/apollo-server/schema/directives/))

Maybe you already know of this, but if not..!

Reply:
Thank you.
I think `model` props will be only URL actually. The props was used for specifying URL but this name makes confusing, so I want to integrate the name from `model` to `url`. But current users are using `model` props so we need to support `model` too in same way with `url`:pray: